pkg/mixin/exec/uninstall.go

```diff
@@ -1,9 +1,6 @@
 package exec
 
 func (m *Mixin) Uninstall(commandFile string) error {
-	err := m.LoadInstruction(commandFile)
-	if err != nil {
-		return err
-	}
-	return m.Execute()
+	// re-use Install's logic
+	return m.Install(commandFile)
```

Comment:
In a follow-on PR, it's up for grabs to anyone (doesn't have to be you!) who would like to extract the logic from `m.Install` into a function that is named a bit more generically so that all of these functions can call into it, maybe `m.ExecuteScript` so that we don't need this comment.

It's a bit confusing to see uninstall calling to install here. I know it's because exec is an odd-ball mixin. 😀